traffic/configure-android.md

```diff
@@ -15,65 +15,57 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
 
-- Install [Fiddler Everywhere v. 1.0.0 and above](https://www.telerik.com/download/fiddler-everywhere).
-- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine.
-Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+- Install the latest version of [Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
-## Configuring Fiddler Everywhere
+- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine. Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+
+## Configuring Fiddler Everywhere Host
 
 1. Enable the remote connections in Fiddler Everywhere through **Settings** > **Connections** > **Allow remote computers to connect**.
 
-1. Check the local IP address of the Fiddler Everywhere application host. You can use [the connection status on the lower right-hand side]({slug connections-section}) to obtain the host IP address. Alternatively, you can obtain the IP address while using  **ipconfig**/ **ifconfig** (depending on the OS). For demonstration purposes, let's assume that the local IP of the Fiddler Everywhere host machine is **192.168.148.39**. 
+1. Check the local IP address of the Fiddler Everywhere application host. You can use [the connection status on the lower right-hand side]({slug connections-section}) to obtain the host IP address. Alternatively, you can obtain the IP address while using terminal command like  **ipconfig**/ **ifconfig** (depending on the OS). For demonstration purposes, let's assume that the local IP of the Fiddler Everywhere host machine is **192.168.100.50**. 
 
 1. Enable [system capturing]({%slug capture-traffic-get-started%}#system-capturing).
 
-## Configuring the Android Device
+## Configuring Android Devices
 
 The following steps apply to real Android devices with access to the Internet through the same network as the Fiddler Everywhere host machine.
 
 1. Set the Fiddler Everywhere proxy on the Android device or emulator.
 
-    1. Open the connected Wi-Fi and tap **Settings**.
-
-    1. Select **Edit** and expand **Advanced Settings**. For previous Android versions, you might have to touch and hold the name of the connected network, then tap **Modify**, and expand **Advanced Settings**.
+    1. Open **Settings** > active Wi-Fi connection > **Proxy**.
 
-    1. On **Proxy**, select **Manual proxy**.
+    1. From the **Proxy** option, select **Manual proxy**.
 
-     - Enter the IP address of the Fiddler Everywhere host&mdash;for example, **192.168.148.39**.
+     - Enter the IP address of the Fiddler Everywhere host&mdash;for example, **192.168.100.50**.
 
      - Enter the Fiddler Everywhere proxy port. By default, the port is **8866**.
 
+     >important Ensure that all VPNs from the Fiddler host and the Android devices are turned off or bypassed.
```

Comment:
We need to check how to support this  - there are cases where devs will work on internal apps, which use APIs part of the internal network (will require VPN).

Reply:
WIll crate a task for that one